src/app/components/tabs/style.module.scss

```diff
@@ -0,0 +1,29 @@
+ .ProjectTitle{
+    text-align: center;
+    margin: 10px  0 !important;
```

Comment:
you use important more in whole app , which is bad practice , so we should talk about that tomorrow , but until that , always check why your style not working and find the cause then treat it , not put !important without known the cause.